x-pack/plugins/security_solution/cypress/tasks/timelines.ts

```diff
@@ -47,7 +47,6 @@ export const openTimeline = (id?: string) => {
 };
 
 export const waitForTimelinesPanelToBeLoaded = () => {
-  cy.get(LOADING_INDICATOR).should('exist');
```

Comment:
If we are removing this line, we should remove line 51 as well. We knew that the loading indicator was present during the loading process and then we were waiting for the element to disappear.

Reply:
@MadameSheema  You are right. Removed next line as well. Changes are here: https://github.com/elastic/kibana/pull/149128/commits/ad000e974eacfce9b3f02d9a69af2c5d3057a6ee